test/editor/Core/BufferTests.re

```diff
@@ -22,6 +22,20 @@ describe("Buffer", ({describe, _}) =>
       validateBuffer(expect, updatedBuffer, [|"a"|]);
     });
 
+    test("BufEnter update does not duplicate content", ({expect}) => {
```

Comment:
Nice! Thanks for adding a test here. Perfect time for a test - now, when we update the algorithm / model for the buffer (like in #80 ) - will have this safety net to verify we don't regress this functionality 👍